test/tools/d_do_test.d

```diff
@@ -580,8 +580,8 @@ bool gatherTestParameters(ref TestArgs testArgs, string input_dir, string input_
     findOutputParameter(file, "GDB_SCRIPT", testArgs.gdbScript, envData.sep);
     findTestParameter(envData, file, "GDB_MATCH", testArgs.gdbMatch);
 
-    if (findTestParameter(envData, file, "POST_SCRIPT", testArgs.postScript))
-        testArgs.postScript = replace(testArgs.postScript, "/", to!string(envData.sep));
+    // if (findTestParameter(envData, file, "POST_SCRIPT", testArgs.postScript))
+    //     testArgs.postScript = replace(testArgs.postScript, "/", to!string(envData.sep));
```

Comment:
Please either remove this completely or add a comment explaining why its commented out

Reply:
In general I would remove, but since this is just a temporary thing... But well, a comment wouldn't hurt actually when I think about it 😁